examples/wnd_outbrain/Outbrain.ipynb

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
This looks like an entirely new notebook (and the previous E2E-nvt.ipynb still also exists in the same notebook).

Can you rename the notebook instead? I'd like to see a diff of the changes, rather than review the entire notebook from scratch.